src/main/java/mate/academy/hibernate/relations/Main.java

```diff
@@ -16,18 +19,35 @@ public static void main(String[] args) {
         SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
 
         Country usa = new Country("USA");
-        CountryService countryService = null; // TODO: initialize this instance
+        Country austria = new Country("AUS");
+        CountryService countryService = new CountryServiceImpl(); // TODO: initialize this instance
         countryService.add(usa);
+        countryService.add(austria);
 
         Actor vinDiesel = new Actor("Vin Diesel");
         vinDiesel.setCountry(usa);
-        ActorService actorService = null; // TODO: initialize this instance
+        Actor arny = new Actor("Arny");
+        arny.setCountry(austria);
+        Actor arny1 = new Actor("Arny");
+        arny1.setCountry(usa);
+        Actor sly = new Actor("Sly");
+        sly.setCountry(usa);
+
+        ActorService actorService = new ActorServiceImpl(); // TODO: initialize this instance
```

Comment:
You should pass the `sessionFactory` to the service implementation constructors. Without the session factory, the services will not be able to perform any operations.